bin/currency/update-currencies.js

```diff
@@ -124,11 +142,14 @@ currencies.forEach(currency => {
 
     if (!SKIP_PUSH) {
       execSync(`git push origin ${branchName} --no-verify`, { cwd });
-      execSync(
-        // eslint-disable-next-line max-len
-        `gh pr create --base main --head ${branchName} --title "[Currency Bot] Bumped ${currency.name} from ${installedVersion} to ${latestVersion}" --body "Tada!"`,
-        { cwd }
-      );
+
+      if (removeBranchIfNoCommits(branchName)) {
```

Comment:
Suggestion: IMO there is no need to push the remote branch if there are no commits.

How about?

> if (hasCommitCount())
>   execSync(`git push origin ${branchName} --no-verify`, { cwd });
>   ...
> else
>   delete local branch

Reply:
Good suggestion ❤️